.github/CHANGELOG.md

```diff
@@ -83,6 +57,245 @@
    2: ──H──╰C────────RZ(0.4)──RZ(-0.4)──RX(0.3)──╰CY──Y──┤
   ```
 
+* A new quantum function transform has been added to push commuting
+  single-qubit gates through controlled operations.
+  [(#1464)](https://github.com/PennyLaneAI/pennylane/pull/1464)
```

Comment:
@antalszava @co9olguy I am wondering if having examples of every compilation transform might be information overload? An alternative is to keep the example for `qml.compile`, but replace the other examples with a condensed bullet list:

```markdown
  The following compilation transforms have been added and are also available
  to use, either independently, or within a `qml.compile` pipeline:

  * `commute_controlled`: push commuting single-qubit gates through controlled operations.
    [(#1464)](https://github.com/PennyLaneAI/pennylane/pull/1464)
    
  * `cancel_inverses`: removes adjacent pairs of operations that cancel out.
    [(#1455)](https://github.com/PennyLaneAI/pennylane/pull/1455)

  * `merge_rotations`: combines adjacent rotation gates of
    the same type into a single gate, including controlled rotations.
    [(#1455)](https://github.com/PennyLaneAI/pennylane/pull/1455)

  * `single_qubit_fusion`: acts on all sequences of
    single-qubit operations in a quantum function, and converts each
    sequence to a single `Rot` gate.
    [(#1458)](https://github.com/PennyLaneAI/pennylane/pull/1458)

  For more details on `qml.compile` and the available compilation transforms, see
  [the compilation documentation](https://pennylane.readthedocs.io/en/latest/code/qml_transforms.html#transforms-for-circuit-compilation).
```

Reply:
yup I agree we can condense the story around compilation